app/controllers/users_controller.rb

```diff
@@ -17,6 +17,6 @@ def create
   private
 
   def user_params
-    params.require(:user).permit(:username, :email, :password, :password_confirmation)
+    params.require(:user).permit(:email, :password)
```

Comment:
Let's see if we can implement all of the existing functionality using Clearance, so our theoretical users don't lose the ability to confirm their password and choose a username